versions/v1.0.0/templates/koordlet-servicemonitor.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    koord-app: koordlet
+  name: koordlet
+  namespace: {{ .Values.installation.namespace }}
+spec:
+  endpoints:
+  - interval: 30s
+    port: http
+    relabelings:
+    - action: replace
+      regex: (.*)
+      replacement: $1
+      sourceLabels:
+      - __meta_kubernetes_pod_node_name
+      targetLabel: node
```

Comment:
I think label `node` is already added in source code of koordlet, do we need to use `relabelings` to do this?

Reply:
Indeed.

In servermonitor we can generate node label for all metrics with just a few lines, I think we can just remove calling and checking `genNodeLabels` in every metric in sourcecode. This is more efficient.